mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/DataImportKafkaHandler.java

```diff
@@ -27,17 +28,18 @@
 public class DataImportKafkaHandler implements AsyncRecordHandler<String, String> {
   private static final Logger LOGGER = LogManager.getLogger();
   static final String RECORD_ID_HEADER = "recordId";
+  public static final String DATA_IMPORT_KAFKA_HANDLER_UUID = "6713adda-72ce-11ec-90d6-0242ac120003";
 
   private Vertx vertx;
   private EventHandlingService eventHandlingService;
-  private KafkaInternalCache kafkaInternalCache;
+  private EventProcessedService eventProcessedService;
 
   public DataImportKafkaHandler(@Autowired Vertx vertx,
                                 @Autowired EventHandlingService eventHandlingService,
-                                @Autowired KafkaInternalCache kafkaInternalCache) {
+                                @Autowired @Qualifier("eventProcessedService") EventProcessedService eventProcessedService) {
```

Comment:
Qualifier annotation is not required here since only 1 implementation of EventProcessedService exists in context

Reply:
@SerhiiNosko Good point. Done.